.github/workflows/ci.yml

```diff
@@ -51,6 +51,9 @@ jobs:
       - name: Run style checks
         uses: ./.github/actions/check_style
 
+      - name: Check unused dependencies
```

Comment:
Let's change the task name, it's not about formatting and spelling for some time already, and this pushes it even further.

`Style Checks`?

Reply:
It looks like this is a separate step that runs after the style checks, rather than being a part of that step.

I think it's okay.